packages/InteractiveIcon/svgs/NavButton/CartFilledBold.jsx

```diff
@@ -45,6 +46,7 @@ CartFilledBold.propTypes = {
       a11yText: PropTypes.string.isRequired,
     }),
   ]).isRequired,
+  variant: PropTypes.oneOf(['default', 'alternative', 'inverted']).isRequired,
```

Comment:
Why do we have alternative here? NavButton does not have an alternative variant.